uswds/uswds-next
# USWDS-Next - POAM: August '24

# Summary

POAM dependency updates for August 2024.

## Vulnerabilities

No changes in vulnerabilities

```node
found 0 vulnerabilities
```

## Testing instructions

1. Run `npm run storybook`
2. Confirm there are no build errors
3. Confirm there are no visual regressions
4. Run prettier and test scripts and confirm there are no issues.

## Dependency updates

| Dependency name | Old version | New version |
| --- | --- | --- |
| @chromatic-com/storybook | ^1.5.0 | ^1.6.1 |
| @rollup/rollup-linux-x64-gnu | ^4.9.5 | ^4.20.0 |
| @storybook/addon-a11y | ^8.1.11 | ^8.2.7 |
| @storybook/addon-essentials | ^8.1.11 | ^8.2.7 |
| @storybook/addon-links | ^8.1.11 | ^8.2.7 |
| @storybook/blocks | ^8.1.11 | ^8.2.7 |
| @storybook/manager-api | ^8.1.11 | ^8.2.7 |
| @storybook/test | ^8.1.11 | ^8.2.7 |
| @storybook/theming | ^8.1.11 | ^8.2.7 |
| @storybook/web-components | ^8.1.11 | ^8.2.7 |
| @storybook/web-components-vite | ^8.1.11 | ^8.2.7 |
| eslint | ^9.5.0 | ^9.8.0 |
| jsdom | ^24.1.0 | ^24.1.1 |
| lit | ^3.1.4 | ^3.2.0 |
| prettier | ^3.3.2 | ^3.3.3 |
| sass | ^1.77.6 | ^1.77.6 |
| storybook | ^8.1.11 | ^8.2.7 |
| vite | ^5.3.1 | ^5.3.5 |
